pom.xml

```diff
@@ -188,6 +188,19 @@
           <artifactId>jacoco-maven-plugin</artifactId>
           <version>0.8.12</version>
         </plugin>
+        <plugin>
```

Comment:
This we will want to remove when we make a PR into dcm4che, I think. But I think we can keep it until we resolve it.